lib/strong_migrations/migration.rb

```diff
@@ -119,6 +119,16 @@ def method_missing(method, *args, &block)
             raise_error :change_column_null,
               code: backfill_code(table, column, default)
           end
+        when :add_foreign_key
```

Comment:
I'm thinking we'll have to reject any call to this method on Rails < 5.2 since `validate: true` is implied.

Reply:
Makes sense.